gateway-provider-identity-assertion-common/pom.xml

```diff
@@ -85,8 +85,23 @@
             <groupId>org.eclipse.jetty</groupId>
             <artifactId>jetty-util</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.shrinkwrap.descriptors</groupId>
+            <artifactId>shrinkwrap-descriptors-api-javaee</artifactId>
+            <scope>compile</scope>
```

Comment:
Don't set scope to compile. Not needed by default. Thats why you don't see `<scope>compile</scope>` anywhere else.

Reply:
So this is just to be in-line with other declaration in the POM file?